openquake/hazardlib/calc/hazard_curve.py

```diff
@@ -208,9 +208,9 @@ def pmap_from_grp(
 
     :returns: a ProbabilityMap instance
     """
-    if source_site_filter == 'SourceSitesFilter':  # default
+    if source_site_filter == 'RtreeFilter':  # default
         source_site_filter = (
-            filters.SourceSitesFilter(maximum_distance)
+            filters.RtreeFilter(sites, maximum_distance, rtree=False)
```

Comment:
It would be more consistent to use always None when rtree is not used or not imported